src/generate/register.rs

```diff
@@ -158,21 +156,21 @@ fn api_docs(
     read_action: Option<ReadAction>,
 ) -> Result<String, std::fmt::Error> {
     fn method(s: &str) -> String {
-        format!("[`{s}`](crate::generic::Reg::{s})")
+        format!("[`{s}`](crate::Reg::{s})")
     }
 
     let mut doc = String::new();
 
     if can_read {
         write!(
             doc,
-            "You can {} this register and get [`{module}::R`]{}. ",
+            "You can {} this register and get [`{module}::R`]{}.",
             method("read"),
             if inmodule { "(R)" } else { "" },
         )?;
 
         if let Some(action) = read_action {
-            doc.push_str("WARN: ");
+            doc.push_str(" WARN: ");
```

Comment:
We could make this a little better with https://github.com/rust-lang/rust/pull/106561 now :D

Reply:
It's interesting. I'll test. The question is should we up MSRV or not?

Reply:
since it's just css it doesn't break when the class doesn't exist. No msrv bump needed

Reply:
Seems like done.
Do you have any other ideas on improving documentation?